src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Action types should be written using ducks pattern as stated in the requirements. The action types should be written in a format like this `const ADD_BOOK = 'appname/reducer/ACTION_TYPE';`. Kindly modify this.